stdlib/public/TensorFlow/CMakeLists.txt

```diff
@@ -60,6 +59,9 @@ if (TENSORFLOW_SWIFT_BINDINGS)
   file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES
     "${TENSORFLOW_SWIFT_BINDINGS}/*.swift")
   list(APPEND SOURCES "${TENSORFLOW_SWIFT_BINDINGS_SOURCES}")
+  file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES_CORE
```

Comment:
I believe you can use a single `file(GLOB_RECURSE ...)`:
```diff
 file(GLOB_RECURSE TENSORFLOW_SWIFT_BINDINGS_SOURCES
  "${TENSORFLOW_SWIFT_BINDINGS}/*.swift"
+ "${TENSORFLOW_SWIFT_BINDINGS}/Core/*.swift")
 list(APPEND SOURCES "${TENSORFLOW_SWIFT_BINDINGS_SOURCES}")
```

Reply:
SGTM. Will fix in the next PR.